docs_user/modules/proc_using-new-subnet-ranges.adoc

```diff
@@ -32,7 +30,7 @@ network_config:
 <1> The new control plane subnet.
 <2> The control plane IP address of the existing data plane node.
 +
```

Comment:
```suggestion

```
I suggest deleting the plus sign and just having a space between line 31 and 33. Currently, line 33 looks like it only applies to line 31, but it applies to the whole step.

Reply:
This didn't work as expected. It's not a single line. I will need to check how to do it properly.

```
The control plane IP address of the existing data plane node. Repeat this configuration for other networks that need to use different subnets for the new and existing parts of the deployment.
```

Reply:
Do you mean that you want it to look like how you pasted it in this comment? If so, it looks okay in the preview now.

Reply:
No I mean the paragraph should belong to the whole list entry, and it should NOT restart numbering. I think the original `+` should be left. This is how continuations are defined in `asciidoc`: https://docs.asciidoctor.org/asciidoc/latest/lists/continuation/#list-continuation